packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -686,6 +687,53 @@ describe('TransactionController', () => {
       );
     });
 
+    it.each([
```

Comment:
To keep this tests tidy, should we add a describe for these such as `with actionId`?

Reply:
Perfect, moved all the tests to a separated describe.